lib/response.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+const debug = require('debug')('hmpo:stubber:response');
+const Base = require('./base');
+
+class Response extends Base {
+    constructor(response, name, basePath) {
+        super(response, name, basePath);
+        debug('New Response', this.name);
+        this.options.repeat = this.options.repeat || 1;
+    }
+
+    send(response, req, res) {
+        if (response.close) {
+            return res.socket.destroy();
+        }
+
+        res.status(response.statusCode || 200);
+
+        let body = this.require(response.body);
+        body = this.value(body, req, res);
+
+        if (typeof body === 'string') {
```

Comment:
Is this to handle sending back HTML pages for doing things like mocking up payment providers?
I guess if we want to use a JSON body, we'd define it in a proper mock?